test/quantization/fx/test_equalize_fx.py

```diff
@@ -238,3 +239,64 @@ def forward(self, x):
             m = M().eval()
             prepared = prepare_fx(m, qconfig_dict, equalization_qconfig_dict=equalization_qconfig_dict)
             self.checkGraphModuleNodes(prepared, expected_node_occurrence=node_occurrence)
+
+    def test_input_weight_equalization_convert(self):
+        """
+        """
+        qconfig_dict = {"": None,
+                        "object_type": [(nn.Linear, default_qconfig),
+                                        (nn.functional.linear, default_qconfig)]}
+
+        default_equalization_qconfig_dict = {
+            "": default_qconfig,
```

Comment:
are we going to use qconfig in equalization_qconfig_dict as well?

Reply:
does this overlap with qconfig_dict?

Reply:
Oops, that should probably be `None`.

Reply:
I mean what does it mean to use {"": default_qconfig} for equalization_qconfig_dict here? would this override the configurations in qconfig_dict? what would happen if we have
qconfig_dict = {"":  None} and equalization_qconfig_dict = {"": default_qconfig}?

Reply:
I think it should not override the `qconfig_dict` configurations. Currently in `node_supports_equalization`, we only prepare the model for equalization if the node is `nn.Linear` or `F.linear`. It would be nice to have a check that all qconfigs in the `equalization_qconfig_dict` use either InputEqualizationObservers or WeightEqualizationObservers, but I wasn't sure how to do so when initializing the `equalization_qconfig_dict`.